using-mobile-extensions/mobile-core/signal/signal-event-reference.md

```diff
@@ -0,0 +1,131 @@
+|                  | analytics.rsids                       | string     | Contains a comma-delimited list of report suites to which the Analytics data should be sent |
+|                  | analytics.server                      | string     | Contains the server endpoint used to collect Analytics data for this customer's report suite |
+| Audience Manager | audience.server                       | string     | Contains the server endpoint used to collect Audience Manager data |
+|                  | audience.timeout                      | number     | Amount of time, in seconds, to wait for a response from Audience Manager before timing out |
+| Campaign         | campaign.pkey                         | string     | App Identifier in the Campaign database                      |
+|                  | campaign.server                       | string     | Contains the server endpoint used to communicate with Campaign |
+| Global           | global.privacy                        | string     | Contains the default setting that the user has request with regards to data privacy |
+|                  | global.ssl                            | boolean    | If set to true, network requests should be dispatched via HTTPS. If false, network requests should be dispatched via HTTP |
+| Identity         | experienceCloud.org                   | string     | Experience Cloud Org Identifier                              |
+|                  | experienceCloud.server                | string     | Custom endpoint to be used for Visitor ID Service network requests |
+|                  | identity.adidEnabled                  | boolean    | If set to true, indicates that the SDK should expect a call to setAdvertisingIdentifier (removed v1.0.1) |
+| Lifecycle        | lifecycle.sessionTimeout              | number     | Number of seconds since the user background the app that must pass in order for the subsequent app launch (or resume) to be considered a new lifecycle session |
+| Rules Engine     | rules.url                             | string[]   | List of Rules endpoints (represented as strings), in order of preference |
+| Target           | target.clientCode                     | string     | Special identifier given to the customer by the Target product. |
+|                  | target.environmentId                  | string     | Indicates the Environment ID for the Target customer         |
+|                  | target.timeout                        | string     | Amount of time, in seconds, to wait for a response from Target before timing out |
+
+#### Event data example
+
+Set Advertising Identifier request
+
+```javascript
+{
+    "global.privacy": "true"
+}
+```
+
+## Events dispatched
+
+The Signal Extension does not dispatch any event.
+
```

Comment:
Let's also mention that Signal extension doesn't share any shared state, section format example https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target/target-event-reference#shared-states